backend/services/project_search_service.py

```diff
@@ -325,7 +328,13 @@ def _filter_projects(search_dto: ProjectSearchDTO, user):
 
         all_results = []
         if not search_dto.omit_map_results:
-            all_results = query.all()
+            query_result = query
+            query_result.column_descriptions.clear()
```

Comment:
Can you please explain about these steps a bit?

Reply:
in this line when we make this query `query.all()`, this query uses all the project attributes and the filter and we don't need all those attributes, it only needs 3. So I use the filter but I only do the query with 3 values.       
These values ​​are used for the geojson of the map
 https://github.com/hotosm/tasking-manager/blob/2e94a550a28b6d9bff2be8953d69ea693fa04b25/backend/services/project_search_service.py#L193